src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -610,7 +610,7 @@ public override void GenerateViewGeometryFromRenderPackagesAndRequestUpdate(IEnu
             OnRequestViewRefresh();
         }
 
-        protected override void DeleteGeometryForIdentifier(string identifier, bool requestUpdate = true)
+        public override void DeleteGeometryForIdentifier(string identifier, bool requestUpdate = true)
```

Comment:
I needed to be able to call this externally. Hope it's okay to break the accessibility here. See PR: https://git.autodesk.com/Dynamo/DynamoPro/pull/178